### PR TITLE
Feature/v4 pypi

### PR DESCRIPTION
**Pull request by hjoaquim**
^

### CODE REVIEW

File: openbb_platform/dev_utils/PUBLISH.md
Comment (piiq) [inline openbb_platform/dev_utils/PUBLISH.md:1]: Do you think the whole content of this folder would be better off in the build folder in the root of the repo? We have a pypi related folder there and this content and code directly relates to that

Comment (hjoaquim) [reply openbb_platform/dev_utils/PUBLISH.md:1]: Yes! I agree

File: .github/workflows/pypi-nightly.yml [hunk 1]
```diff
@@ -12,37 +12,52 @@ jobs:
...
+          --sdist
+          --wheel
+          --outdir dist/
+          .
+
+      - name: OpenBBTerminal - Publish distribution 📦 to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages-dir: dist/
+          password: ${{ secrets.TEST_PYPI_API_TOKEN_OBB_TERMINAL_NIGHTLY }}
```
Comment (hjoaquim) [inline .github/workflows/pypi-nightly.yml:47]: @piiq we need to create an `openbb-terminal` project on PyPI and then create a project scoped API token and add it to the GitHub secrets.

Comment (hjoaquim) [reply .github/workflows/pypi-nightly.yml:47]: cc @luqmanbello

**Comment by hjoaquim** [comment]
@luqmanbello just tagging you to ask if you see any problems with the changes on the actions

**Pull request merged**